Update logistics language about virtual and in person, and pandemic (#268)
oceanhackweek/oceanhackweek.github.io: ohw23/logistics/index.md

# Logistics Overview

> Some of you will participate in person at different locations and others will participate virtually from time zones all over the world. Each of these gathering styles has strengths and weaknesses. We are committed to providing an engaging and full experience for everyone and will strive to facilitate cross communication and collaboration across all participants.

Author: ```suggestion
Some of you will participate in person in Seattle or Perth and others will participate virtually from time zones all over the world. Each of these gathering styles has strengths and weaknesses. We are committed to providing an engaging and full experience for everyone and will strive to facilitate cross communication and collaboration across all participants.
```
